composer.json

```diff
@@ -217,7 +217,7 @@
         "su-sws/nobots": "^8.2",
         "su-sws/react_paragraphs": "^8.2",
         "su-sws/stanford_actions": "^8.2",
-        "su-sws/stanford_fields": "dev-D8CORE-6288",
+        "su-sws/stanford_fields": "^8.1",
```

Comment:
We will either need to do a release on that module or set this to `8.x-dev`

Reply:
I'll just set it to the dev version for now.